basic_autonomy/src/basic_autonomy.cpp

```diff
@@ -75,9 +75,34 @@ namespace basic_autonomy
             std::vector<PointSpeedPair> points_and_target_speeds;
 
             cav_msgs::LaneFollowingManeuver lane_following_maneuver = maneuver.lane_following_maneuver;
-            
+
             auto lanelets = wm->getLaneletsBetween(starting_downtrack, lane_following_maneuver.end_dist + detailed_config.buffer_ending_downtrack, true, true);
 
+
+            bool lanelets_defined = !maneuver.lane_following_maneuver.lane_ids.empty();
+            ROS_DEBUG_STREAM("lanelets_defined: " << lanelets_defined);
+            bool isFromPlatooning = maneuver.lane_following_maneuver.parameters.planning_strategic_plugin == "PlatooningStrategicIHPPlugin";
+            ROS_DEBUG_STREAM("isFromPlatooning: " << isFromPlatooning);
+
+            if (lanelets_defined && isFromPlatooning)
```

Comment:
Per discussion on slack this plugin specific solution is not desirable. Since the general solution was already written lets try evaluating that first.

Reply:
Per discussion with Sudhakar we will wait on this and you should add a comment/issue like in my above comment